flang/lib/Lower/OpenMP.cpp

```diff
@@ -29,31 +29,42 @@ getDesignatorNameIfDataRef(const Fortran::parser::Designator &designator) {
   return dataRef ? std::get_if<Fortran::parser::Name>(&dataRef->u) : nullptr;
 }
 
+template <typename T>
+static void createPrivateVarSyms(Fortran::lower::AbstractConverter &converter,
+                                 const T *clause) {
+  Fortran::semantics::Symbol *sym = nullptr;
+  const Fortran::parser::OmpObjectList &ompObjectList = clause->v;
+  for (const auto &ompObject : ompObjectList.v) {
+    std::visit(
+        Fortran::common::visitors{
+            [&](const Fortran::parser::Designator &designator) {
+              if (const auto *name = getDesignatorNameIfDataRef(designator)) {
+                sym = name->symbol;
+              }
+            },
+            [&](const Fortran::parser::Name &name) { sym = name.symbol; }},
+        ompObject.u);
+    constexpr bool init =
+        std::is_same_v<T, Fortran::parser::OmpClause::Firstprivate>;
+    [[maybe_unused]] bool success =
+        converter.createHostAssociateVarClone(*sym, init);
+    assert(success && "Privatisation failed due to existing binding");
```

Comment:
Does LLVM have any preference as far as American vs. British English spelling?

Reply:
Thanks, I think there is a preference for American. Have switched to Privatization.